app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt

```diff
@@ -75,8 +75,8 @@ object ApplicationModule {
     }
     private val MIGRATION_3_4 = object : Migration(3, 4) {
         override fun migrate(database: SupportSQLiteDatabase) {
-            database.execSQL("ALTER TABLE CHILD_DATA ADD COLUMN voiceAnalysisOption  TEXT NOT NULL DEFAULT ${VoiceAnalysisOption.MachineLearning.name}")
-            database.execSQL("ALTER TABLE CLIENT_DATA ADD COLUMN voiceAnalysisOption TEXT NOT NULL DEFAULT ${VoiceAnalysisOption.MachineLearning.name}")
+            database.execSQL("ALTER TABLE CHILD_DATA ADD COLUMN voiceAnalysisOption  TEXT NOT NULL DEFAULT ${VoiceAnalysisOption.MACHINE_LEARNING.name}")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/dd68c2ed2c957a6e8bc85bec93f0f874e01caa7c/app/src/main/kotlin/co/netguru/baby/monitor/client/application/di/ApplicationModule.kt#L78"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>